manageiq-operator/pkg/controller/manageiq/manageiq_controller.go

```diff
@@ -117,6 +138,12 @@ func (r *ReconcileManageiq) Reconcile(request reconcile.Request) (reconcile.Resu
 }
 
 func (r *ReconcileManageiq) generateHttpdResources(cr *miqv1alpha1.Manageiq) error {
+
+	httpdDeployment := miqtool.NewHttpdDeployment(cr)
+	if err := r.createk8sResIfNotExist(cr, httpdDeployment, &appsv1.Deployment{}); err != nil {
+		return err
+	}
```

Comment:
Why was this moved? It seems like the config maps should be created before the deployment.